setup.py

```diff
@@ -70,7 +70,7 @@ def _get_readme():
                         'importlib==1.0.3',
                         'passlib==1.6.2',
                         'pexpect==3.3',
-                        'requests==2.4.1',
+                        'requests>=2.4.1,<3.0',
```

Comment:
It doesn't exist yet, but see <http://semver.org/>

Maybe worth having a comment about that in general.